cli/cli.py

```diff
@@ -21,10 +31,51 @@
 
 
 def list_resources(provider: str, resource_type: str):
+    print_banner(False)
```

Comment:
Since this is not being done in the current CLI let's remove it.
```suggestion
```

Reply:
Done